<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>目标检测v1.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>951</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_VedioShow">
     <attribute name="title">
      <string>目标检测界面</string>
     </attribute>
     <widget class="QLabel" name="label_ShowImg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>640</width>
        <height>480</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string>视频显示</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_data_info">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>90</y>
        <width>271</width>
        <height>401</height>
       </rect>
      </property>
      <property name="title">
       <string>检测结果</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>34</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>目标位置:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>目标移动速度:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_target_number">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>340</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>xxx</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>340</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>目标个数:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>目标大小:</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEditSpeed">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>250</y>
         <width>161</width>
         <height>81</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEditLocation">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>161</width>
         <height>81</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEditSize">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>150</y>
         <width>161</width>
         <height>81</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_target_detect_time">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>370</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>xxx</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>370</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>单帧检测时间:</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_button">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>10</y>
        <width>271</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>控制按钮</string>
      </property>
      <widget class="QPushButton" name="pushButtonEndDetection">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>结束检测</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonOutputData">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>数据导出</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonBeginDetection">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>开始检测</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>其他</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>966</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
